textproc/stork/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo   1.0
+PortGroup           github  1.0
+
+github.setup        jameslittle230 stork v1.5.0
```

The format of `github.setup` is `{author} {project} {version} {version_tag_prefix}`, so this would look like:

```suggestion
github.setup        jameslittle230 stork 1.5.0 v
```